### PR TITLE
Create ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by bode

Ticket: [DM-3546](https://cumulocity.atlassian.net/browse/DM-3546)
Original PR: [6283](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6283)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Replace device as plugin [GRAFT][release/cd]
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Replace device as plugin
```

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3546
+version: 1019.24.0
+---
+Replace device as plugin [GRAFT][release/cd]
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the context of enhancing device management capabilities, the Cumulocity IoT platform now supports replacing devices using a plugin. This change allows users to seamlessly replace an existing device with a new one while preserving the historical data, alarms, and other associated information. The impact of this improvement is a more efficient device management process, enabling users to maintain continuity and access to valuable device insights even when physical devices need to be replaced.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:17]:
```suggestion
The Replace device feature is now delivered as a self-hosted plugin by the Device management application. This allows that the feature is easily removable from the application if not needed and that it is plugged to other applications with All devices list or Subassets grid.

Additionally, WebSDK hooks hookDataGridActionControls and hookService have been introduced.
```

**Review by dafkk — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3546
+version: 1019.24.0
+---
+The Replace device feature is now delivered as a self-hosted plugin by the Device management application. This allows that the feature is easily removable from the application if not needed and that it is plugged to other applications with All devices list or Subassets grid.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:17]:
```suggestion
The "Replace device" functionality is now delivered as a self-hosted plugin by the Device management application. This way, the functionality can easily be removed from the application if not needed and plugged to other applications with an **All devices** list or **Subassets** view.
```

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+date: ""
+title: Replace device as plugin
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:3]:
```suggestion
title: Replace device functionality available as plugin
```

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3546
+version: 1019.24.0
+---
+The Replace device feature is now delivered as a self-hosted plugin by the Device management application. This allows that the feature is easily removable from the application if not needed and that it is plugged to other applications with All devices list or Subassets grid.
+
+Additionally, WebSDK hooks hookDataGridActionControls and hookService have been introduced.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:19]: This should actually be a separate entry. Is this part of the same ticket/change?

Comment (bozhidardedov) [reply content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:19]: It was all merged with a single PR, so technically it is part of a single change.

Comment (BeateRixen) [reply content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md]: @janhommes What is your take here? Do we need a separate entry for the Web SDK changes? I don´t think that it makes sense to combine this in one change even if it was one PR?

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3546
+version: 1019.24.0
+---
+The "Replace device" functionality is now delivered as a self-hosted plugin by the Device management application. This way, the functionality can easily be removed from the application if not needed.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:17]:
```suggestion
The "Replace device" functionality is now delivered as a self-hosted plugin by the Device Management application. This way, the functionality can easily be removed from the application if not needed.
```

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md (hunk 6 above)
Comment (janhommes) [reply content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md]: I am fine with having this in one entry (as it is feature relevant, not Web SDK) but I would phrase it more customer friendly like "Additionally, the device replace wizard was made more extensible by introducing hookable interfaces.

File: content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3546
+version: 1019.24.0
+---
+The "Replace device" functionality is now delivered as a self-hosted plugin by the Device Management application. This way, the functionality can easily be removed from the application if not needed.
+
+Additionally, WebSDK hooks hookDataGridActionControls and hookService have been introduced.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-24-0-replace-device-as-plugin-graft-release-cd.md:19]:
```suggestion
Additionally, the device replace wizard was made more extensible by introducing hookable interfaces (`hookDataGridActionControls` and `hookService`).
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**